### PR TITLE
add freelance jobposting

### PR DESCRIPTION
**Pull request by sixhobbits**
add freelance job post faq

### CODE REVIEW

File: faqs/howtojobpost-freelance.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+layout: page
+title: How do I post a freelance job opportunity?
+permalink: /faqs/howtojobpost-freelance
+---
+
+To post a job opportunity in #freelance-jobpostings you need to use the "Post a Job" workflow. To do this click the ⚡️ icon next to the input bar and fill in the form.
+
+![Workflow freelance jobposting](/images/faq/freelance-job-posting-workflow.png)
+
+Once the form has been submitted a request will be sent to the admins for approval.
```
Comment (JohnRoux) [inline faqs/howtojobpost-freelance.markdown]: "to an approval team"

**Review by JohnRoux — COMMENTED** [review]
Looks pretty solid 👍

File: faqs/howtojobpost-freelance.markdown (hunk 1 above)
Comment (JohnRoux) [inline faqs/howtojobpost-freelance.markdown]:
```suggestion
Once the form has been submitted a request will be sent to the approval team.
```

**Review by JohnRoux — CHANGES_REQUESTED** [review]
Added Suggestion

**Comment by sixhobbits** [comment]
made the same change in the one I copied it from.

**Review by n1c — APPROVED** [review]

**Review by JohnRoux — APPROVED** [review]

**Pull request merged**